src/storage/src/error.rs

```diff
@@ -240,6 +240,17 @@ pub enum Error {
         location: Location,
     },
 
+    #[snafu(display(
+        "Failed to delete WAL namespace, region id: {}, source: {}",
+        region_id,
+        source
+    ))]
+    DeleteWalNamespace {
+        region_id: RegionId,
+        #[snafu(backtrace)]
```

Comment:
This backtrace way is outdated, use location instead
```suggestion
        location: Location
```